lib/IRGen/IRGenModule.h

```diff
@@ -511,14 +511,10 @@ class IRGenModule {
   llvm::IntegerType *Int1Ty;           /// i1
   llvm::IntegerType *Int8Ty;           /// i8
   llvm::IntegerType *Int16Ty;          /// i16
-  union {
-    llvm::IntegerType *Int32Ty;          /// i32
-    llvm::IntegerType *RelativeAddressTy;
-  };
-  union {
-    llvm::PointerType *Int32PtrTy;            /// i32 *
-    llvm::PointerType *RelativeAddressPtrTy;
-  };
+  llvm::IntegerType *Int32Ty;          /// i32
+  llvm::IntegerType *RelativeAddressTy;
+  llvm::PointerType *Int32PtrTy;            /// i32 *
```

Comment:
The comments should be vertically aligned with the other comments if possible, and please put the pointer types next to the pointee types.